TaffoInitializer/IndirectCallPatcher.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TAFFO_INDIRECTCALLPATCHER_H
+#define TAFFO_INDIRECTCALLPATCHER_H
+
+#include "llvm/IR/Dominators.h"
```

Comment:
```suggestion
#include <llvm/IR/Dominators.h>
```

You may also want to re-run a `clang-format` after this change

Reply:
Hahaha I disagree with @skeru, you should use "", it was already fine as it was :P

Reply:
Follow @danielecattaneo 's suggestions here